src/main/resources/public/ts/directives/external-calendar-form/external-calendar-form.ts

```diff
@@ -50,11 +57,19 @@ class Controller implements IViewModel {
     }
 
     isFormComplete(): boolean {
-        return !!this.$scope.vm.calendar && !!this.$scope.vm.calendar.title && !!this.$scope.vm.calendar.icsLink;
+        return !!this.$scope.vm.calendar && !!this.$scope.vm.calendar.title && (!!this.$scope.vm.calendar.icsLink !== !!this.$scope.vm.calendar.platform);
     }
 
-    translate(text: string): string {
-        return lang.translate(text);
+
+    changeSelection(field: string, platform?: string): void {
+        switch (field) {
```

Comment:
this.$scope.vm.calendar[field] = undefined;

ça prend moins de place 😉

Reply:
vu avec @qpic-cgi